### PR TITLE
feat(admin): show backend version

### PR DESCRIPTION
**Pull request by mattjoke**
* the footer contains information about latest backend version

### CODE REVIEW

File: libs/perun/components/src/lib/perun-footer/perun-footer.component.ts [hunk 1]
```diff
@@ -55,14 +56,19 @@ export class PerunFooterComponent implements OnInit {
...
     private translateService: TranslateService,
+    private utilsService: UtilsService,
     private dialog: MatDialog
   ) {}
 
   ngOnInit(): void {
     this.translateService.onLangChange.subscribe((lang) => {
       this.language = lang.lang;
     });
-    this.version = require('../../../../../../package.json').version as string;
```
Comment (xflord) [inline libs/perun/components/src/lib/perun-footer/perun-footer.component.ts:65]: It's a bit confusing that now we display the version of back-end, however the link in the footer still leads to GUI releases. I would probably update the version in package.json and then display both the version of GUI and BE.

Comment (mattjoke) [reply libs/perun/components/src/lib/perun-footer/perun-footer.component.ts:65]: Yeah, you're right about the links. The following commit addresses this; the footer will now display both information (i.e. GUI and BE version) and their respective links

**Review by xflord — COMMENTED** [review]
Good job, a few more visual details - I think the footer would look better without spaces between 'v.' and the version numbers. Also, 'RPC' instead of 'Backend' is a better fit, I believe. What do @HejdaJakub and @bodnara think?

File: libs/perun/components/src/lib/perun-footer/perun-footer.component.html [hunk 2]
```diff
@@ -62,13 +62,21 @@ <h6 [ngStyle]="{'color': headersTextColor}" class="font-weight-bold">
         >{{index ? ', ' + item.name : item.name}}</a
       >
     </span>
-    Version
+    , GUI: v.
```
Comment (xkureck) [inline libs/perun/components/src/lib/perun-footer/perun-footer.component.html:65]:
```suggestion
    , web application: v.
```

File: libs/perun/components/src/lib/perun-footer/perun-footer.component.html [hunk 3]
```diff
@@ -62,13 +62,21 @@ <h6 [ngStyle]="{'color': headersTextColor}" class="font-weight-bold">
...
     <a
       [href]="githubRepository"
       [ngStyle]="{'color': copyrightTextColor}"
       class="text-decoration-none"
       target="_blank"
-      >{{version}}</a
+      >{{guiVersion}}</a
+    >
+    , Backend: v.
```
Comment (xkureck) [inline libs/perun/components/src/lib/perun-footer/perun-footer.component.html:73]:
```suggestion
    , core: v.
```

**Review by xkureck — COMMENTED** [review]
more user friendly, users dont know what gui or backend is

File: libs/perun/components/src/lib/perun-footer/perun-footer.component.html (hunk 3 above)
Comment (HejdaJakub) [inline libs/perun/components/src/lib/perun-footer/perun-footer.component.html:73]: There are suggestions like `Backend`,  `RPC` or `core`, so I also would like to suggest my option. I am not sure what is commonly used across applications, but the `server` is used at least in our old GUI, so our users should be familiar with it.
```suggestion
    , server: v.
```

File: libs/perun/components/src/lib/perun-footer/perun-footer.component.ts [hunk 4]
```diff
@@ -46,23 +47,37 @@ export class PerunFooterComponent implements OnInit {
...
+
+    this.guiVersion = require('../../../../../../package.json').version as string;
+    this.utilsService.getPerunStatus().subscribe((val) => {
+      const versionString = val[0];
+      this.backendVersion = versionString.substring(versionString.indexOf(':') + 2);
+      this.backendVersion = '17.2.0';
+      this.guiVersion = '17.2.0';
```
Comment (HejdaJakub) [inline libs/perun/components/src/lib/perun-footer/perun-footer.component.ts:78]: You probably forgot to delete these two lines.
```suggestion
```

Comment (mattjoke) [reply libs/perun/components/src/lib/perun-footer/perun-footer.component.ts]: Yes, thank you for pointing it out.

**Review by HejdaJakub — APPROVED** [review]

**Pull request merged**